rtcdelta.py

```diff
@@ -29,22 +31,50 @@ def ensure_path_exists(path):
     if not os.path.exists(path):
         os.mkdir(path)
 
+
+def get_changes(changesets):
+    if not changesets:
+        return
+
+    for changeset in changesets:
+        print("Traversing %s" % changeset)
+        for change in changeset.getChanges():
+            print("Extracting before&after for change: %s" % change.comment)
+            change.fetchBeforeStateFile(arguments["--beforePath"])
+            change.fetchAfterStateFile(arguments["--afterPath"])
+
+
+def get_children(workitem):
+    print("Getting children_list of %s : %s"% (workitem.identifier, workitem.title))
+    workitem_list = [workitem]
+    children_list = workitem.getChildren()
+    if not children_list:
+        return workitem_list
+
+    for children in children_list:
+        workitem_list += get_children(children)
+
+    return workitem_list
+
+
 if __name__ == '__main__':
     arguments = docopt(__doc__, version='RTC Delta')
     if arguments["--logs"]:
         setup_basic_logging()
+    else:
+        logging.basicConfig(level=logging.CRITICAL)
+
     rtc_client = RTCClient(arguments["--url"],
                           arguments["--user"],
                           arguments["--pass"],
                           ends_with_jazz=arguments["--endsWithJazz"])
     ensure_path_exists(arguments["--beforePath"])
     ensure_path_exists(arguments["--afterPath"])
-    work_item = rtc_client.getWorkitem(arguments["--workitem"]) # ,returned_properties="dc.ientifier,dc:title,rtc_cm:com.ibm.team.filesystem.workitems.change_set.com.ibm.team.scm.ChangeSet"
-    print("%s : %s"% (work_item.identifier, work_item.title))
-    changesets = work_item.getChangeSets()
-    for changeset in changesets:
-        print("Traversing %s" % changeset)
-        for change in changeset.getChanges():
-            print("Extracting before&after for change: %s" % change.comment)
-            change.fetchBeforeStateFile(arguments["--beforePath"])
-            change.fetchAfterStateFile(arguments["--afterPath"])
\ No newline at end of file
+    workitem = rtc_client.getWorkitem(arguments["--workitem"]) # ,returned_properties="dc.ientifier,dc:title,rtc_cm:com.ibm.team.filesystem.workitems.change_set.com.ibm.team.scm.ChangeSet"
+    if arguments['--children']:
+        for children in get_children(workitem):
```

Comment:
singular of children is child, pls change to:

 for child in get_children(workitem):